WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -262,55 +293,69 @@ def export_setJobApplicationStatus( self, jobID, appStatus, source = 'Unknown' )
       new_status = status
     minorStatus = result['Value']['MinorStatus']
 
-    result = jobDB.setJobStatus( int( jobID ), new_status, application = appStatus )
+    result = jobDB.setJobStatus(int(jobID), new_status, application=appStatus)
     if not result['OK']:
       return result
 
-    result = logDB.addLoggingRecord( int( jobID ), new_status, minorStatus, appStatus, source = source )
+    result = logDB.addLoggingRecord(int(jobID), new_status, minorStatus, appStatus, source=source)
     return result
 
   ###########################################################################
   types_setJobParameter = [[basestring, int, long], basestring, basestring]
-  def export_setJobParameter( self, jobID, name, value ):
+
+  def export_setJobParameter(self, jobID, name, value):
     """ Set arbitrary parameter specified by name/value pair
         for job specified by its JobId
     """
 
-    result = jobDB.setJobParameter( int( jobID ), name, value )
+    if elasticJobDB:
+      result = elasticJobDB.setJobParameter(int(jobID), name, value)
+    else:
+      result = jobDB.setJobParameter(int(jobID), name, value)
```

Comment:
No, this is not correct: we may write in both, writing in one does not exclude from writing in the other.